server/serializers.py

```diff
@@ -130,14 +130,14 @@ class WorkflowSerializer(serializers.ModelSerializer):
     owner_name = serializers.SerializerMethodField()
 
     def get_read_only(self, obj):
-        return obj.read_only(self.context['user'])
+        return obj.read_only(self.context.get('user', False))
```

Comment:
Why is the False required? Could use a comment

Reply:
In case we have a request with no user, will add a comment